src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Context/VolleyManagementEntities.cs

```diff
@@ -2,9 +2,12 @@
 
 namespace VolleyManagement.Data.MsSql.Context
 {
+    using System;
+    using System.Configuration;
     using System.Data.Entity;
     using System.Data.Entity.ModelConfiguration.Conventions;
     using Entities;
+    using Microsoft.Extensions.Configuration;
```

Comment:
What are those changes for?

Reply:
Seems like leftovers from development experiments. I will remove.